fruitify/models.py

```diff
@@ -75,8 +75,9 @@ def training_step(self, batch: Tuple[Tensor, Tensor], batch_idx: int) -> Tensor:
         """
         # TODO: use F.cross_entropy() to compute the loss, and return it. Help: examples/cross_entropy.py
         X, y = batch
+        criterion = F.cross_entropy()
         S_subword = self.forward(X)
-        loss = ...
+        loss = criterion(S_subword, y)
```

Comment:
<img width="833" alt="image" src="https://user-images.githubusercontent.com/56193069/121547583-3a07a380-ca47-11eb-8e6f-23e1db326f87.png">

용택님! PR 감사합니다.  다만 sigma^N_i=1 을 빠트리신 것 같습니다!
그리고 F.cross_entropy()를 쓰시는 경우, `criterion`으로 instantiate하지말고,  functional 패러다임에 맞게 함수로 바로 사용하는게 나을 것 같아요!
```suggestion
        loss = F.cross_entropy(S_subword, y)   # (N, K,|S|) -> (N,)
        loss = loss.sum()  # (N, K, |S|) -> (1,) 
```

Reply:
resolved with 92e74fe